source/stylesheets/modules/_cookie-banner.scss

```diff
@@ -0,0 +1,10 @@
+.cookie-banner-text {
+  color: black;
+  margin-top: 0px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.cookie-banner {
+  background-color: lightblue;
```

Comment:
`lightblue` should be a colour variable, eg: `$light-blue`. Colour vars are declared in https://github.com/alphagov/govwifi-product-page/blob/master/source/stylesheets/util/_colours.scss. Or use a predefined design system variable.